### DISCUSSION TITLE
not able to read iceberg table stored in azure data lake

### DISCUSSION THREAD

**Discussion by rajasekarv** [StarRocks/starrocks#12695] [Announcements]
I am trying to read external iceberg table stored in azure blob storage. I have added hadoop-azure-datalake-3.3.0.jar and hadoop-azure-3.3.0.jar in Hadoop path inside both fe and in be/lib/hadoop/common and in be/lib/hadoop/hdfs directory.

It is able to read the metadata from hive table in fe but throws an error in be

W1031 03:42:36.130653 62060 pipeline_driver_executor.cpp:133] [Driver] Process error, query_id=0f4216e3-58ce-11ed-a626-9a451fd27c85, instance_id=0f4216e3-58ce-11ed-a626-9a451fd27c86, status=Not supported: No FileSystem associated with abfss://*@*.dfs.core.windows.net/*/*/data/00010-2574-b3be9ca5-15eb-4edf-8111-a975ead16d9a-00001.parquet

**Comment by alberttwong** [comment]
I would try the latest version to see if this fixes it.
